src/components/WalletUnlockDialog/index.jsx

```diff
@@ -23,22 +23,30 @@ const messages = defineMessages({
   },
 });
 
-class WalletUnlockDialog extends React.Component {
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state) => ({
+  walletUnlockDialogVisibility: state.App.get('walletUnlockDialogVisibility'),
+}), (dispatch) => ({
+  toggleWalletUnlockDialog: (isVisible) => dispatch(appActions.toggleWalletUnlockDialog(isVisible)),
+  unlockWallet: (passphrase, timeout) => dispatch(appActions.unlockWallet(passphrase, timeout)),
+}))
+
+export default class WalletUnlockDialog extends React.Component {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     walletUnlockDialogVisibility: PropTypes.bool.isRequired,
     toggleWalletUnlockDialog: PropTypes.func.isRequired,
+    unlockWallet: PropTypes.func.isRequired,
     intl: intlShape.isRequired, // eslint-disable-line react/no-typos
   };
 
-  static defaultProps = {
-  };
-
   constructor(props) {
     super(props);
 
     this.state = {
-      unlockMinutes: 1,
+      passphrase: '',
+      unlockMinutes: 5,
```

Comment:
let's use a const for the default value?

Reply:
added a default unlockWalletMins in `config`